homeassistant/components/rachio/manifest.json

```diff
@@ -2,7 +2,17 @@
   "domain": "rachio",
   "name": "Rachio",
   "documentation": "https://www.home-assistant.io/integrations/rachio",
-  "requirements": ["rachiopy==0.1.3"],
-  "dependencies": ["http"],
-  "codeowners": []
+  "requirements": [
+    "rachiopy==0.1.3"
+  ],
+  "dependencies": [
+    "http"
+  ],
+  "codeowners": [],
```

Comment:
You want to claim it?

Reply:
Might as well as I have quite a few of these

Reply:
done

Reply:
well on the wrong branch.. tired.. fixing

Reply:
done for real